### ISSUE TITLE
Dark color scheme for accessibility

### ISSUE THREAD

**Issue by ekaitz-zarraga** [ProtonMail/WebClients#173]
Protonmail only supports a light colorscheme. This can be a problem for users with dark color schemes in their OS. Opening a light website can hurt you in your eyes.

Adding a dark-color scheme like the one at:
https://amdelamar.com/pm-theme/
Can help those users who prefer to use their screens with a darker setup.

There are many ways to support this. As Protonmail WebClient uses JS there's no problem to add a light/dark switch like other sites like youtube or twitter do.
Also, in order to capture the colorscheme preference of users there's a CSS mediaquery available:
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

Implementing this officially would make users able to choose without having to install any third party CSS that is a feature many of the users won't know how to deal with.

The main issue I see on this is with the rendering of HTML emails with custom colors, but they can be left as they come and try to implement the dark mode only on plain-text emails.

**Comment by darigovresearch** [comment]
Think this issue is a very important one with respect to accessibility. Similar issues have been raised in https://github.com/ProtonMail/proton-mail-android/issues/32 and think that this should be made as a priority for precisely those reasons if possible.

In the web client it would just need a switcher in the settings even though you can copy & paste your own css currently but this functionality should not be considered as a final solution

**Comment by nebeker** [comment]
This is solved in Version 4.0.0 Beta 41, via the theme switcher. Settings gear - Theme. I'm using Carbon, which has a very dark blue background and lighter blue accents.

An additional "Dark Proton" theme could be added and made the default when the system reports a dark theme.
